### DISCUSSION TITLE
Do you know a way to get Flex for new installs?

### DISCUSSION THREAD

**Discussion by azorantezana** [airsdk/Adobe-Runtime-Support#3196] [General]
I tried to download Flex SDK Installer 3.3.2 in this link https://flex.apache.org/installer.html
but it is not possible to install this Flex SDK Installer 3.3.2 in an executable version. Do you know other way to install flex (step by step or with a recent video step by step).

**Comment by ajwfrost** [comment]
I don't believe it's possible to get hold of any Flex SDK versions that were published by Adobe; the only one still available for download afaik is the Apache Flex SDK, 4.16.1
https://flex.apache.org/

**Reply by Nisaba** [reply]
Hi,
The thing is that we can't use the Flex installer :
![image](https://github.com/airsdk/Adobe-Runtime-Support/assets/34550856/da4ce645-0dd6-4405-b218-ffc4701b1cbf)

Only direct links work.
i didn't find active support in the  website. It seems that the Flex project is not alive anymore ?

**Reply by hydroper** [reply]
Do you really need the installer though? Apache Royale is the successor of Flex, so probably worthy giving it a look (but I'm not sure whether Royale works well with latest AIR).

**Reply by joshtynjala** [reply]
A while back, I compiled a list of direct .zip download links on Stack Overflow to old versions of the Adobe Flex SDK (from 3.0.0 to 4.6.0). It appears that they still work, for now.

https://stackoverflow.com/a/21269691/10768

**Reply by ajwfrost** [reply]
Thanks Josh - I didn't know those ones were there! That webserver itself (fpdownload.adobe.com) will stay around for a while I think because it's used for Flex component deployments i.e. for apps that didn't link everything at build time, when they're first run they need to get at the core Flex RSLs. So, hopefully those zip files will continue to be hosted there too...

**Comment by joshtynjala** [comment]
I recently wrote a tutorial on by blog about installing Apache Flex with Adobe AIR from HARMAN:

https://joshblog.net/2024/how-to-install-apache-flex-with-adobe-air-from-harman/

**Reply by FlashTang** [reply]
> I recently wrote a tutorial on by blog about installing Apache Flex with Adobe AIR from HARMAN:
> 
> https://joshblog.net/2024/how-to-install-apache-flex-with-adobe-air-from-harman/

Thank you joshtynjala
